app/controllers/doorkeeper/authorized_applications_controller.rb

```diff
@@ -1,6 +1,7 @@
 module Doorkeeper
   class AuthorizedApplicationsController < Doorkeeper::ApplicationController
     before_filter :authenticate_resource_owner!
+    layout 'doorkeeper/admin'
```

Comment:
`current_resource_owner` is not the admin. We shouldn't be showing this layout here.